### DISCUSSION TITLE
Installing Linux OBS NDI Plug-in on Steam Deck (SteamOS Desktop)

### DISCUSSION THREAD

**Discussion by zerocoregithub** [DistroAV/DistroAV#820] [Q&A - Support]
Hi everyone,

I'm trying to setup my Steam Deck as the receiving NDI device to process footage coming from my PC. I already have OBS itself fully working on the Deck, however Linux is a bit foreign to me. As far as I understand it the Linux version of OBS NDI here on github is compiled for Ubuntu, which I can't seem to launch as-is via SteamOS desktop.

Is there a way to still get the OBS NDI installed on SteamOS? Any support would be very much appreciated.

**Comment by TOCA2FREAK** [comment]
Hi,

I too am trying to install this plugin onto my Steam Deck.  I hear steam deck runs on Arch Linux.  Does anyone know how to install this plugin on this OS?

Many thanks. ;)

**Comment by TOCA2FREAK** [comment]
I assume we will never get the help we need with this. :(

**Comment by Trouffman** [comment]
This is not a supported option yet, but some options are happening at : https://github.com/obs-ndi/obs-ndi/issues/724#
until then, it's hard to provide help/direction

**Comment by Trouffman** [comment]
This is working via : https://flathub.org/apps/com.obsproject.Studio.Plugin.DistroAV
